src/core/proto/neighbour.cpp

```diff
@@ -50,6 +50,7 @@
 #include <netinet/icmp6.h>
 
 #define MODULE_NAME "ne"
+DOCA_LOG_REGISTER(ne);
```

Comment:
Why not do DOCA_LOG_REGISTER(MODULE_NAME) in all files??

Reply:
DOCA_LOG_REGISTER expects an expression - not a string.

Maybe I could have used macro magic to convert string to expression (remove the quotes)

Not sure it's possible, but I think it's better to keep it simple and not introduce magics neverthelss.

wdyt?